README.md

```diff
@@ -6,4 +6,40 @@ Copyright 2023 LunarG, Inc.
 SPDX-License-Identifier: Apache-2.0
 -->
 
-# Vulkan-Utility-Libraries
\ No newline at end of file
+# Vulkan-Utility-Libraries
+
+This repo was created to share code across various Vulkan repositories, solving long standing issues for Vulkan 
+SDK developers and users.
+
+## Historical Context
+
+For a while the `Vulkan-Validation-Layers` was the primary mechansim of code sharing for various Vulkan repos.
+This was done out of neccessity/history.
+
+This made the `Vulkan-Validation-Layers` install source code that was unreliable and subject to breaking changes.
+Which would frequently cause issues for package managers, SDK packaging, Vulkan developers, and validation layer developers.
+
+This repo was created to facilitate official source deliverables that can be used by developers reliably.
```

Comment:
```suggestion
The `Vulkan-ValidationLayers` contained many libraries and utilities that were useful for other Vulkan repositories, and became the primary mechanism for code sharing in the Vulkan ecosystem.

This caused the `Vulkan-ValidationLayers` to have to maintain and export source code which was never intended for that purpose. This not only hindered development of the Vulkan-ValidationLayers, but would frequently break anyone depending on the source code due to the poorly located nature of it.

This repository was created to facilitate official source deliverables that can be reliably used by developers.
```

I'm still not super happy with how this paragraph reads. It is trying to convey a LOT of information very succinctly. So take this as more of a suggestion than a 'use my paragraph' instead.

Reply:
Agreed. I'm trying not to go into too much detail. I like your suggestions though.